packages/@tailwindcss-upgrade/src/migrate-postcss.ts

```diff
@@ -0,0 +1,95 @@
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { pkg } from './utils/packages'
+import { info, success, warn } from './utils/renderer'
+
+// Migrates simple PostCSS setups. This is to cover non-dynamic config files
+// similar to the ones we have all over our docs:
+//
+// ```js
+// module.exports = {
+//  plugins: {
+//    tailwindcss: {},
+//    autoprefixer: {},
+//  }
+// }
+export async function migratePostCSSConfig(base: string) {
+  let configPath = await detectConfigPath(base)
+  if (configPath === null) {
+    // TODO: We can look for an eventual config inside package.json
```

Comment:
https://sourcegraph.com/search?q=context:global+file:package.json+%27%22postcss%22:+%7B%27+tailwindcss&patternType=keyword&sm=0 let's support it